src/test/java/guitests/BoardTests.java

```diff
@@ -277,15 +279,15 @@ public void boards_panelCount_newBoardCreated() {
         PanelControl panelControl = ui.getPanelControl();
 
         traverseMenu("Boards", "New");
-        waitUntilNodeAppears(hasText("OK"));
-        clickOn("OK");
+        waitUntilNodeAppears(hasText("No"));
+        clickOn("No");
```

Comment:
The prompt is actually something like `Do you want to save the current board before open a new one?` instead of `Open a new board anyway?`. In other words, no matter whether `Yes` or `No` is chosen, a new board will be created. Just the matter of whether the current board is saved.